TombEngine/Specific/setup.cpp

```diff
@@ -45,254 +44,6 @@ void InitialiseGameFlags()
-		obj->collision = ObjectCollision;
-		obj->control = SmashObjectControl;
-	}
-
-	obj = &Objects[ID_SMASH_OBJECT5];
-	if (obj->loaded)
-	{
-		obj->initialise = InitialiseSmashObject;
-		obj->collision = ObjectCollision;
-		obj->control = SmashObjectControl;
-	}
-
-	obj = &Objects[ID_SMASH_OBJECT6];
-	if (obj->loaded)
-	{
-		obj->initialise = InitialiseSmashObject;
-		obj->collision = ObjectCollision;
-		obj->control = SmashObjectControl;
-	}
-
-	obj = &Objects[ID_SMASH_OBJECT7];
-	if (obj->loaded)
-	{
-		obj->initialise = InitialiseSmashObject;
-		obj->collision = ObjectCollision;
-		obj->control = SmashObjectControl;
-	}
-
-	obj = &Objects[ID_SMASH_OBJECT8];
-	if (obj->loaded)
-	{
-		obj->initialise = InitialiseSmashObject;
-		obj->collision = ObjectCollision;
-		obj->control = SmashObjectControl;
-	}
-
-	obj = &Objects[ID_CRUMBLING_FLOOR];
-	if (obj->loaded)
-	{
-		obj->initialise = InitialiseFallingBlock;
-		obj->collision = FallingBlockCollision;
-		obj->control = FallingBlockControl;
-	}
-
-	for (int objNum = ID_KEY_HOLE1; objNum <= ID_KEY_HOLE16; objNum++)
-	{
-		InitKeyHole(obj, objNum);
-	}
-
-	for (int objNum = ID_PUZZLE_HOLE1; objNum <= ID_PUZZLE_HOLE16; objNum++)
-	{
-		InitPuzzleHole(obj, objNum);
-	}
-
-	for (int objNum = ID_PUZZLE_DONE1; objNum <= ID_PUZZLE_DONE16; objNum++)
-	{
-		InitPuzzleDone(obj, objNum);
-	}
-	
-	for (int objNum = ID_ANIMATING1; objNum <= ID_ANIMATING128; objNum++)
-	{
-		InitAnimating(obj, objNum);
-	}
-
-	InitAnimating(obj, ID_LASERHEAD_BASE);
-	InitAnimating(obj, ID_LASERHEAD_TENTACLE);
-
-	obj = &Objects[ID_TIGHT_ROPE];
-	if (obj->loaded)
-	{
-		obj->initialise = InitialiseTightrope;
-		obj->collision = TightropeCollision;
-		obj->drawRoutine = nullptr;
-		
-		obj->usingDrawAnimatingItem = false;
-	}
-
-	obj = &Objects[ID_PARALLEL_BARS];
-	if (obj->loaded)
-	{
-		obj->collision = HorizontalBarCollision;
-	}
-
-	obj = &Objects[ID_EARTHQUAKE];
-	if (obj->loaded)
-	{
-		obj->drawRoutine = nullptr;
-	}
-
-	obj = &Objects[ID_HIGH_OBJECT2];
-	if (obj->loaded)
-	{
-		obj->drawRoutine = nullptr;
-		obj->control = HighObject2Control;
-	}
-
-	obj = &Objects[ID_LENS_FLARE];
-	if (obj->loaded)
-	{
-		//obj->drawRoutine = DrawLensFlare;
-		
-	}
-
-	obj = &Objects[ID_WATERFALLMIST];
-	if (obj->loaded)
-	{
-		obj->control = ControlWaterfallMist;
-		obj->drawRoutine = nullptr;
-	}
-
-	for (int objNum = ID_WATERFALL1; objNum <= ID_WATERFALL6; objNum++)
-	{
-		obj = &Objects[objNum];
-		if (obj->loaded)
-		{
-			obj->control = nullptr;
-		}
-	}
-
-	obj = &Objects[ID_WATERFALLSS1];
-	if (obj->loaded)
-	{
-		obj->control = nullptr;
-	}
-
-	obj = &Objects[ID_WATERFALLSS2];
-	if (obj->loaded)
-	{
-		obj->control = nullptr;
-	}
-
-	for (int objNum = ID_SHOOT_SWITCH1; objNum <= ID_SHOOT_SWITCH4; objNum++)
-	{
-		obj = &Objects[objNum];
-		if (obj->loaded)
-		{
-			obj->initialise = InitialiseShootSwitch;
-			obj->control = ControlAnimatingSlots;
-			obj->collision = ShootSwitchCollision;
-		}
-	}
-}
-
-void TrapObjects()
```

Comment:
Where this function was moved to? I don't see this code reintroduced somewhere else.

Reply:
To tr5_objects.cpp